extract_wheels/lib/bazel.py

```diff
@@ -20,6 +21,8 @@ def generate_build_file_contents(name: str, dependencies: List[str]) -> str:
     there may be no Python sources whatsoever (e.g. packages written in Cython: like `pymssql`).
     """
 
+    data_exclude = ["**/*.py", "**/* *", "BUILD", "WORKSPACE"] + pip_data_exclude
```

Comment:
Why are we excluding `"**/* *"`? IIUC this means any file with a space in the name, not sure we want that.

Reply:
There are several reasons for that :)

1. I only refactored this definition, it was also in the previous version: https://github.com/dillon-giacoppo/rules_python_external/blob/master/extract_wheels/lib/bazel.py#L32
2. Afair it originates in rules_python, files with spaces in their names are also excluded in the BUILD files there
3. If you have files in the pip package (it happens quite frequently) with a space in their name then Bazel will fail when you try to build, at least on Linux this is the behavior. Bazel does not allow for files with spaces in their names, hence we need to protect from such. In fact also directories with spaces in their name may cause issues, but with the pip_data_exclude option this can be tackled by the user (we may want to add that as a default exclude if you think).

Reply:
Thanks for the detailed response, appreciate it!